src/server/enterprise/cmds/cmds.go

```diff
@@ -124,11 +125,29 @@ func RegisterCmd() *cobra.Command {
 			}
 			defer ec.Close()
 
+			if pachdUsrAddr == "" {
+				pachdUsrAddr = c.GetAddress()
```

Comment:
`GetAddress` actually doesn't include the protocol (`grpc` versus `grpcs`). I'm looking at fixing our client package to make these cases easier to handle, can you add a TODO here and I can fix this once I'm done with the client package?

Reply:
This fix is actually in master, if you have the time to rebase? `GetAddress` now returns `grpcutil.PachdAddress` which has a method to get the fully-qualified address.